app/components/tab_navigation_component.html.erb

```diff
@@ -1,14 +1,25 @@
 <%= content_tag(:nav, aria: { label: }, **tag_options, class: [*tag_options[:class], 'tab-navigation']) do %>
   <ul class="usa-button-group usa-button-group--segmented">
     <% routes.each do |route| %>
-      <li class="usa-button-group__item">
-        <%= render ButtonComponent.new(
-              url: route[:path],
-              big: true,
-              outline: !current_path?(route[:path]),
-              aria: { current: current_path?(route[:path]) ? 'page' : nil },
-            ).with_content(route[:text]) %>
-      </li>
+      <% if  route[:event_name] %>
+        <%=  render ClickObserverComponent.new(event_name: route[:event_name], class: "usa-button-group__item") do %>
```

Comment:
If it's not easily possible to use `<li>` here per the failing accessibility test, maybe we can assign `role="list-item"`?